AppServices/CustomerAppService.cs

```diff
@@ -39,24 +39,24 @@ public bool Update(int id, Customer CustomerUpdated)
             return _customerServices.Update(id, CustomerUpdated);
         }
 
-        public List<Customer> GetById(int id)
+        public List<Customer> SearchId(int id)
         {
-            return _customerServices.GetById(id);
+            return _customerServices.GetAll(x => x.Id == id);
         }
 
-        public List<Customer> SearchFullName(string FullName)
+        public List<Customer> SearchFullName(string fullName)
         {
-            return _customerServices.SearchFullName(FullName);
+            return GetAll(x => x.FullName.Equals(fullName));
         }
 
-        public List<Customer> SearchEmail(string Email)
+        public List<Customer> SearchEmail(string email)
```

Comment:
Usar Get é o padrão de nomenclatura para método de filtro e quando pegar todos para aquele filtro é obrigatório especificar o All após o Get

```suggestion
        public List<Customer> GetAllByEmail(string email)
```